lib/grid.rb

```diff
@@ -0,0 +1,25 @@
+class Grid
+
+  COLUMNS = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+  ROWS = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J"]
+
+  def cells
+    cells = []
+    COLUMNS.each do |number|
+      cell = ""
+      ROWS.each do |letter|
+        cell += number.to_s
+        cell += letter
+        cells.push(cell)
+        cell = ""
+      end
```

Comment:
You can change line 10 to line 16 to one line like so:

ROWS.each do |letter| cells.push(number.to_s + letter) end